packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-services/TerraService.tool.mdx

```diff
@@ -62,6 +62,29 @@ export.config = {
 };
 ```
 
+### selector
+
+An optional selector can be configured to specify the element to be captured in the screenshot. The default selector is `data-terra-test-content`. Accessibility testing ignores this option and always tests the entire document.
+
+Type: `string`
+
+Required: `false`
+
+Default: `[data-terra-test-content] *:first-child`
```

Comment:
With the current implementation the default value for the service is actually `undefined` because the value is set in the `wdio.conf`, but a different `wdio.conf` or one that accidentally overwrite `serviceOptions` using the Terra service would end up with an undefined selector

Reply:
This [update](https://github.com/cerner/terra-toolkit/pull/464/commits/04718224294139afe8b92e05a16b2910ba550b35) now defaults the selector in the service.